html5lib/tests/tree_construction.py

```diff
@@ -51,14 +51,17 @@ def runtest(self):
         fragmentContainer = self.test['document-fragment']
         expected = self.test['document']
         expectedErrors = self.test['errors'].split("\n") if self.test['errors'] else []
+        script = True
+        if 'script-off' in self.test:
+            script = False
```

Comment:
Right. But we're never going to pass anything containing `<script>document.write("<p>")</script>`, hence it makes sense to skip it. Script elements are only going to be in the tests if they mutate the DOM.